src/components/Table/components/Table.tsx

```diff
@@ -41,6 +46,7 @@ const Table: FC<TableProps> = ({
     };
 
     return (
+        // @ts-expect-error table props (height, width) conflict with HeightProps and WidthProps from styled-system
```

Comment:
I think it should be `ts-ignore`

Reply:
should we update TS to 3.9 instead?

Reply:
https://devblogs.microsoft.com/typescript/announcing-typescript-3-9/#ts-expect-error-comments

Reply:
I'll do it in a follow-up